extension/src/cli/dvc/discovery.ts

```diff
@@ -229,12 +239,13 @@ export const recheckGlobal = async (
     return
   }
 
-  const version = await setup.getCliVersion(cwd, true)
-  const cliCompatible = isVersionCompatible(version)
-  const isCompatible = isCliCompatible(cliCompatible)
+  let isCompatible = await checkVersion(setup, cwd, undefined)
+  if (!isCompatible) {
+    isCompatible = await checkVersion(setup, cwd, true)
```

Comment:
[Q] Should we put another delay in here between the global and venv checks? Run them with an offset?

Reply:
I think it might just make this slower when installing.